{{ cookiecutter.__project_name_kebab }}/.github/workflows/test.yml

```diff
@@ -67,20 +69,19 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9']
-        django: ['3.2']
-        wagtail: ['2.15', '4.0', '4.1']
+        python: ['3.8', '3.9', '3.10', '3.11']
+        django: ['3.2', '4.0', '4.1', '4.2']
+        wagtail: ['4.1', '4.2', '5.0', '5.1']
```

Comment:
it is a pity we have the tox config, and then this. We should follow up with setup for https://github.com/ymyzk/tox-gh-actions/ so that we only run tox in CI

Reply:
I've made an issue for this https://github.com/wagtail/cookiecutter-wagtail-package/issues/55